shared/teams/index.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import Render from './render'
+import pausableConnect from '../util/pausable-connect'
+import openURL from '../util/open-url'
+
+const mapStateToProps = state => ({})
+
+type DispatchProps = {
+  onCreateTeam: () => void,
+  onJoinTeam: () => void,
+  onReadDoc: () => void,
+}
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  onCreateTeam: () => {
+    // TODO: Hook this up. Need to change onShowNewTeamDialog to
+    // make its conversationIDKey parameter optional first.
+    console.log('onCreateTeam not implemented yet')
+  },
+  onJoinTeam: () => {
+    // TODO: Hook this up once we have a join team dialog.
+    console.log('onJoinTeam not implemented yet')
+  },
+  onReadDoc: () => {
+    openURL('https://keybase.io/docs/command_line/teams_alpha')
+  },
+})
+
+const mergeProps = (stateProps, dispatchProps: DispatchProps) => ({
```

Comment:
we dont need the mergeProps if its just going to basically do the default, i'd just remove it

Reply:
Done. Turns out what I wanted was just to type the return value of `mapDispatchToProps`.